src/test/java/sysc4806/graduateAdmissions/model/OperationTest.java

```diff
@@ -0,0 +1,28 @@
+package sysc4806.graduateAdmissions.model;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+/**
+ * Tests for the Operation enumeration. These tests check that an enum instance can
+ * be successfully created, and that the descriptions for each enum are correct.
+ *
+ * @author luke
+ */
+class OperationTest {
+    @Test
+    public void createEnumSuccessfully(){
```

Comment:
I'm fine with no javadoc for the test methods in this PR, but I'll make the comment anyway for the record since I'd usually like to see javadoc.

No change required.

Reply:
I don't want to set a precedent for not writing comments on things, so I will do that